beacon-chain/sync/validate_beacon_attestation.go

```diff
@@ -73,6 +75,32 @@ func (s *Service) validateCommitteeIndexBeaconAttestation(ctx context.Context, p
 		return pubsub.ValidationReject
 	}
 
+	if featureconfig.Get().EnableSlasher {
+		// Feed the indexed attestation to slasher if enabled. This action
+		// is done in the background to avoid adding more load to this critical code path.
+		go func() {
+			preState, err := s.chain.AttestationPreState(ctx, att)
```

Comment:
Note: This is a likely a dos vector. Someone could easily trick a slasher node to generate a state that uses more resources